biz.aQute.remote/src/aQute/remote/api/packageinfo

```diff
@@ -1 +1 @@
-version 1.1.0
\ No newline at end of file
+version 2.0.0
```

Comment:
Should this really be a version 2 bump?

Reply:
Nope, we can make it a ProviderType. All known implementers are in our build.